radiacode-examples/webserver.html

```diff
@@ -1,141 +1,197 @@
 <!DOCTYPE html>
-<html>
```

Comment:
Have you changed anything in this file? If it's only a format change, I would prefer not to do it in this MR as it's not related to "Bluetooth on all platforms"